react.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     "plugin:react-hooks/recommended",
     "plugin:testing-library/react",
     "plugin:cypress/recommended",
-    "prettier/react",
+    "plugin:prettier/recommended",
```

Comment:
this does not look like it's the same thing as before

Reply:
yes they merged all rules -> https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21

Reply:
either you add `prettier` or `plugin:prettier/recommended` as a plugin. Both is ok according to this changelog

Reply:
Frankly, I think I can remove the whole line since we extend `index.js` and we already have the line in there. Will remove it.

Reply:
we have non-react projects, so it makes sense to only include React-related stuff here and keeping them out of the index instead (not sure this fully applies)

Reply:
you'll be able to judge better what to do. thanks for the confirmation :)